### PR TITLE
zpaq: update test file's sha

### PR DESCRIPTION
**Pull request by mitchblank**
It looks like the author repacked this test file.  It's only used for testing puruposes, the code itself has not changed.

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
license declaration:
```

// zpaq.cpp - Journaling incremental deduplicating archiver

#define ZPAQ_VERSION "7.15"
/*
  This software is provided as-is, with no warranty.
  I, Matt Mahoney, release this software into
  the public domain.   This applies worldwide.
  In some countries this may not be legally possible; if so:
  I grant anyone the right to use this software for any purpose,
  without any conditions, unless such conditions are required by law.
zpaq is a journaling (append-only) archiver for incremental backups.
Files are added only when the last-modified date has changed. Both the old
and new versions are saved. You can extract from old versions of the
archive by specifying a date or version number. zpaq supports 5
compression levels, deduplication, AES-256 encryption, and multi-threading
using an open, self-describing format for backward and forward
compatibility in Windows and Linux. See zpaq.pod for usage.
```

File: Formula/zpaq.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Zpaq < Formula
   url "http://mattmahoney.net/dc/zpaq715.zip"
   version "7.15"
   sha256 "e85ec2529eb0ba22ceaeabd461e55357ef099b80f61c14f377b429ea3d49d418"
+  license "Unlicense"
```
Comment (chenrui333) [inline Formula/zpaq.rb:7]:
```suggestion
  license :public_domain
```

cc @Rylan12

Comment (mitchblank) [reply Formula/zpaq.rb:7]: I picked "Unlicense" based on this file `COPYING` which includes:
```
For more information, please refer to <http://unlicense.org>
```

Comment (Rylan12) [reply Formula/zpaq.rb:7]: I'm getting mixed messages from the readme/homepage/`COPYING`. First, note that `:public_domain` is not the same as [`Unlicense`](https://spdx.org/licenses/Unlicense.html) (which is a valid open-source license).

From the [homepage](http://mattmahoney.net/dc/zpaq.html):

> zpaq is written by Matt Mahoney and released to the public domain. It includes code from libdivsufsort 2.0 (C) Yuta Mori, 2003-2008, MIT license, public domain code for AES from libtomcrypt by Tom St Denis and public domain code for salsa20 by D. J. Bernstein

This indicates `all_of: [:public_domain, "MIT"]`

From the `readme`:

> Code from divsufsort.c is embedded in libzpaq.cpp.
> divsufsort.c is (C) 2003-2008 Yuta Mori, MIT license (see source code).
> It is also available from libdivsufsort-lite 2.0 from
> http://code.google.com/p/libdivsufsort/
> All remaining code is public domain. See COPYING.

This indicates `all_of: [:public_domain, "MIT"]`. However, the `readme` also says:

> ```
> File            Ver.   Description
> -----------     ----   -----------
> COPYING                Unlicense.
> ```

The `COPYING` file contains the [Unlicense license text](https://spdx.org/licenses/Unlicense.html) and the [MIT license text](https://spdx.org/licenses/MIT.html).

Note that while the `Unlicense` text includes the line `This is free and unencumbered software released into the public domain.`, it does not mean that `:public_domain` is accurate.

---

So it seems that the correct license is either:

```ruby
license all_of: [:public_domain, "MIT"]
```

or

```ruby
license all_of: ["Unlicense", "MIT"]
```

**Pull request merged**